certified-connectors/JG Integrations/Readme.md

```diff
@@ -0,0 +1,14 @@
+## JG Integrations Flow Connector
```

Comment:
@jglassonnz Please follow the [template provided for the readme.md file](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) and update your connector's readme.md file accordingly with the required information.